config/initializers/geocoder.rb

```diff
@@ -16,7 +16,7 @@ def language=(_locale); end
   Geocoder::Lookup::Test.set_default_stub(
     [
       { 'city' => '', 'country' => 'United States', 'state_code' => '' },
-    ]
+    ],
```

Comment:
Dear god can we remove this rubocop rule or at least talk about it? ADDING a dangling comma seems like a really worthless change to require another commit and another run of the pipeline.

Reply:
> Dear god can we remove this rubocop rule or at least talk about it? ADDING a dangling comma seems like a really worthless change to require another commit and another run of the pipeline.

It definitely caught me by surprise, but moreso the fact that I didn't change anything which didn't already exist 😅  So I guess my problem is less about whether or not we do the dangling comma, and more that our code isn't consistently enforced by linting.

I suppose #4055 will aim to address this?